lib/features/family/features/parent_giving_flow/presentation/pages/organization_list_family_page.dart

```diff
@@ -0,0 +1,194 @@
+// ignore_for_file: no_default_cases
```

Comment:
For the filename: In most of the places we use the UK spelling or at least in the backend we tried to do that consistently. So 'organisation' instead of 'organization'. 
What do we wanna do here? :)

Reply:
I love UK spelling so I will adjust.